src/Components/Register/index.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import { createTable } from '../App';
+import './register.css';
+
+const firstRowTable = ['Product', 'Type', 'Locatie', 'Leverancier'];
+const query = [['printer 1', 'printer', '5G', 'bol.com'], ['desktop 1', 'desktop', '4G', 'amazon'], ['scanner 1', 'scanner', '3G', 'coolblue']];
```

Comment:
Umm ik neem aan dat je hier de resultaten van een query bedoeld (meestal zou ik de variabele iets als data noemen),
 verder zal de data er wel even iets anders uitzien. Zie dit voor een indicatie:
```javascript
const products = [
  {
    name: 'printer 1',
    category: { name: 'printer' },
    location: '5G',
    supplier: { name: 'bol.com' }
  }, // 1 product
]
```

Reply:
Ah yes, ik probeerde te anticiperen op dat hier dus idd een query zou zijn die ik uitvoer waar een resultaat uit komt die ik vervolgens in de functie prop